docs/troubleshooting.rst

```diff
@@ -284,6 +284,36 @@ to clear counters on firewall rulesets or single rules
   vyos@vyos:~$ clear firewall ipv6-name <ipv6 ruleset name> counters
   vyos@vyos:~$ clear firewall ipv6-name <ipv6 ruleset name> rule <rule#> counters
 
+Comment command
+---------------
+The :cfgcmd:`comment` command allows you to add comments above the current section. 
+The :cfgcmd:`comment` command cannot be used at the top level of the configuration 
```

Comment:
Since when it cannot be used at the top level? ;)
```
vyos@vyos-test-2# comment firewall "The thingie that keeps bad packets out of your router, you know"
[edit]
vyos@vyos-test-2# commit
[edit]
vyos@vyos-test-2# show 
 /* The thingie that keeps bad packets out of your router, you know */
 firewall {
     all-ping enable
```

It's much more important (and true) that comments cannot be added to individual values of multi nodes, like addresses.

Reply:
Will fix.  Just transposed from existing documentation.